src/components/SplitButton/SplitButton.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { IMenuItemProps, Menu, MenuItem, Position } from '@blueprintjs/core'
+import cx from 'classnames'
+import React, { MouseEvent } from 'react'
+import { Button, ButtonProps, Popover } from '../..'
+import css from './SplitButton.css'
+
+type SplitButtonProps = Omit<ButtonProps, 'rightIcon'>
```

Comment:
Component doesn't accept right icon and uses additional button to open a dropdown